composer.json

```diff
@@ -36,6 +36,7 @@
         "symfony/console": "^2.7 || ^3.3 || ^4.0",
         "symfony/dependency-injection": "^2.7 || ^3.3 || ^4.0",
         "symfony/filesystem": "^2.7 || ^3.3 || ^4.0",
+        "symfony/lock": "^3.4 || ^4.1",
```

Comment:
Is version `^4.1` required, can this be lowered to `^4.0` to be in line with other Symfony dependencies?

Reply:
yep, true.